java/code/src/com/suse/manager/webui/controllers/admin/AdminViewsController.java

```diff
@@ -93,7 +108,7 @@ public static void initRoutes(JadeTemplateEngine jade) {
     public static ModelAndView showMonitoring(Request request, Response response, User user) {
         Map<String, Object> data = new HashMap<>();
         data.put("isUyuni", ConfigDefaults.get().isUyuni());
-        return new ModelAndView(data, "controllers/admin/templates/monitoring.jade");
+        return new ModelAndView(data, "controllers/admin/templates/issv3/monitoring.jade");
```

Comment:
maybe better use "hub" instead of "issv3" ?

Reply:
Mmm, let me double check this I don't think we should have changed it.